lib/similarity_utils.py

```diff
@@ -21,3 +22,70 @@ def iou(lista1: List[Any], lista2: List[Any]) -> float:
         intersection = set(lista1).intersection(set(lista2))
         union = set(lista1).union(set(lista2))
         return len(intersection) / len(union)
+
+
+def sort_two_lists(list1: List[Any], list2: List[Any], reverse: bool = True):
```

Comment:
To bardzo nienaturalny sposób sortowania. Napisz mi jaki dla Ciebie ma cel ta funkcja a podpowiem jak ją zrobić lepiej

Reply:
Chciałem posortować dwie listy równocześnie. Jedna byłaby posortowana od najmniejszej do największej, a druga lista powtórzyła kroki z sortowania listy pierwszej.

Reply:
Np. żeby `sort([5, 2, 4, 3, 1], ['Adam', 'Zosia', 'Krystyna', 'Karol', 'Tomek']` zwracał listy:
`[1, 2, 3, 4, 5]` oraz `['Tomek', 'Zosia', 'Karol', 'Krystyna', 'Adam']`

Reply:
okey - sam kod metody bym w tej sytuacji zrobił podobnie. Jest to dość nietypowe bo nie trzyma się dwóch list w osobnych zmiennych jeśli łączą je jakieś zależności. Coś czuje, że twoje dwie listy tak naprawdę są dictem, albo pd.Series, które chciałbyś posortować po wartości. Na pewno natomiast w takiej sytuacji metoda ta powinna mieć asercję na odpowiadające rozmiary list i doc-stringa z opisem działania - ten element uzupełnij koniecznie.

Reply:
Docstring został poprawiony. Dodałem w nim przykład działania oraz moją próbę opisu działania metody w słowach.
Jest też assert na długości list